src/server/db/database.router.js

```diff
@@ -1,15 +1,23 @@
-import express from 'express';
+// @flow
+
+import * as express from 'express';
+
+import Database from './database';
 
 class DatabaseRouter {
-  constructor(database) {
+  router: express.Router<>;
+
+  database: any;
+
+  constructor(database: Database) {
     this.router = express.Router();
     this.database = database;
-    this.router.get('/getUserFunctions', (req, res) => this.getUserComponentsRequest(req, res));
+    this.router.get('/getUserComponents', (req, res) => this.getUserComponentsRequest(req, res));
```

Comment:
Like the rename here. Is there a way to factor route names out so they aren't repeated in the frontend/backend

Reply:
What do you mean by this?